### PR TITLE
Roberta docstring reworking

### PR DESCRIPTION
**Pull request by abuelnasr0**
#867
link to colab used for testing: https://colab.research.google.com/drive/1TR4dCo_15IOaMdWZq1rrYcK4mscW5ACS#scrollTo=Iy8I5mZpYeMz

### CODE REVIEW

File: keras_nlp/models/roberta/roberta_preprocessor.py [hunk 1]
```diff
@@ -75,76 +62,76 @@ class RobertaPreprocessor(Preprocessor):
...
     # Tokenize and pack a single sentence.
-    sentence = tf.constant(" afternoon sun")
-    preprocessor(sentence)
-    # Same output.
-    preprocessor(" afternoon sun")
-
-    # Tokenize and a batch of single sentences.
-    sentences = tf.constant(
-        [" afternoon sun", " night moon"]
+    preprocessor("The quick b  rown fox jumped.")
```
Comment (chenmoneygithub) [inline keras_nlp/models/roberta/roberta_preprocessor.py:83]: is space in "b  rown" intended?

File: keras_nlp/models/roberta/roberta_preprocessor.py [hunk 2]
```diff
@@ -75,76 +62,76 @@ class RobertaPreprocessor(Preprocessor):
...
+
+    # Tokenize a batch of single sentences .
```
Comment (chenmoneygithub) [inline keras_nlp/models/roberta/roberta_preprocessor.py:85]: nit: remove space between sentences and period.

File: keras_nlp/models/roberta/roberta_preprocessor.py [hunk 3]
```diff
@@ -75,76 +62,76 @@ class RobertaPreprocessor(Preprocessor):
...
+    preprocessor(["The quick brown fox jumped.", "Call me Ishmael."])
+
+    # Preprocess a batch of sentence pairs.
+    # When handling multiple sequences, always convert to tensors first !
```
Comment (chenmoneygithub) [inline keras_nlp/models/roberta/roberta_preprocessor.py:89]: same, remove the extra space

File: keras_nlp/models/roberta/roberta_tokenizer.py [hunk 4]
```diff
@@ -43,48 +43,39 @@ class RobertaTokenizer(BytePairTokenizer):
     `tf.Tensor` with static shape `[None]`.
 
     Args:
-        vocabulary: string or dict, maps token to integer ids. If it is a
-            string, it should be the file path to a json file.
-        merges: string or list, contains the merge rule. If it is a string,
-            it should be the file path to merge rules. The merge rule file
-            should have one merge rule per line. Every merge rule contains
-            merge entities separated by a space.
+        vocabulary: A dictionary or a string filename path. If passing a
```
Comment (chenmoneygithub) [inline keras_nlp/models/roberta/roberta_tokenizer.py:46]: Suggested edit: 
```
A dictionary mapping tokens to integer ids, or file path to a json file containing the token to id mapping. 
```

File: keras_nlp/models/roberta/roberta_tokenizer.py [hunk 5]
```diff
@@ -43,48 +43,39 @@ class RobertaTokenizer(BytePairTokenizer):
...
+            filename, the file should be a json file. both json and dictionary
+            should map a single word piece token string to an integer id.
+        merges: A list of merge rules or a string filename path, If passing a
```
Comment (chenmoneygithub) [inline keras_nlp/models/roberta/roberta_tokenizer.py:49]: nit: filename path => file path, and use "." to replace ","

File: keras_nlp/models/roberta/roberta_tokenizer.py [hunk 6]
```diff
@@ -43,48 +43,39 @@ class RobertaTokenizer(BytePairTokenizer):
...
+            filename, the file should have one merge rule per line. Every merge
+            rule contains merge entities separated by a space.
```
Comment (chenmoneygithub) [inline keras_nlp/models/roberta/roberta_tokenizer.py:51]: We could add an example to make it more clear.

**Review by chenmoneygithub — CHANGES_REQUESTED** [review]
Thanks for the PR!

Overall LGTM! Left some minor comments on the style.

File: keras_nlp/models/roberta/roberta_tokenizer.py (hunk 6 above)
Comment (abuelnasr0) [reply keras_nlp/models/roberta/roberta_tokenizer.py:51]: there is a custom vocabulary example just a few lines later containing an example and passing vocabs and merges to the tokenizer.

**Review by chenmoneygithub — APPROVED** [review]
Thanks, LGTM!

One minor thing - in the future after you fix a comment, you may click on "resolve" button so that we know it's been addressed.

**Pull request merged**